.github/workflows/release.yaml

```diff
@@ -102,13 +108,13 @@ jobs:
       - name: Upload rbuilder artifact
         uses: actions/upload-artifact@v4
         with:
-          name: rbuilder-${{ matrix.configs.target }}${{ matrix.features && '-' }}${{ matrix.features }}
+          name: rbuilder-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.configs.target }}${{ matrix.features && '-' }}${{ matrix.features }}
```

Comment:
this results in such filenames:

```
rbuilder-workflows-release-version-6725aee-aarch64-unknown-linux-gnu-redact-sensitive
```

why is the `workflows-release-version` useful? this seems to be added by this change 🤔

Reply:
no strong feelings. I dropped it, [updated run here](https://github.com/flashbots/rbuilder/actions/runs/12169439042).